ios/.gitignore

```diff
@@ -0,0 +1,32 @@
+*.mode1v3
```

Comment:
What are those files? If they present only on your local machine, just remove them rather than including in here

Reply:
I did not add them, flutter added them, so I am checking them in... I did not manually modify git ignore at all